client/app/assets/less/inc/base.less

```diff
@@ -244,18 +244,14 @@ text.slicetext {
   display: flex;
   align-items: center;
 
-  h3 {
-    margin-right: 5px !important;
-  }
-
   .label {
     margin-top: 3px;
     display: inline-block;
   }
 
   .favorites-control {
     font-size: 19px;
-    margin-right: 5px;
+    margin-right: 10px;
```

Comment:
This also affects the DashboardPage now, we added those extra 5px on the Query Pages now, I thought it should also be in there for consistence, LMK if you think it differently.